src/game/Chat/Chat.h

```diff
@@ -282,6 +282,7 @@ class ChatHandler
         bool HandlePartyBotUnpauseCommand(char * args);
         bool HandlePartyBotUnequipCommand(char * args);
         bool HandlePartyBotRemoveCommand(char * args);
+        bool HandlePartyBotRepairitemsCommand(char * args);
```

Comment:
It should probably be called `HandlePartyBotRepairItemsCommand` instead (the `I` of `Items` should be uppercased).